packages/cli/src/ui/components/DebugProfiler.tsx

```diff
@@ -47,25 +48,25 @@ export const profiler = {
   },
 
   reportFrameRendered() {
+    if (this.profilersActive === 0) {
+      return;
+    }
     const now = Date.now();
-    // Simple frame detection logic (a write after at least 16ms is a new frame)
```

Comment:
this hacky logic depended on stdout as in't needed now that we get actual render events.